src/components/MapSelect/control/LegendControl.test.js

```diff
@@ -4,7 +4,6 @@ import LegendControl from './LegendControl';
 describe('Leaflet legend control', () => {
   let mapDiv;
 
-  // eslint-disable-next-line unicorn/no-object-as-default-parameter
```

Comment:
Any particular reason why this commented out line was removed? It prevented the linter from tripping. Either fix the issue the linter is reporting or comment it out
🤷

Reply:
The linter told me that `unicorn/no-object-as-default-parameter` didn't exist.  After removing it, the liter didn't complain.

Reply:
Even after bringing branch up to date with `develop` and running `npm i`?

Reply:
I see, it was only locally. After `npm i` the problem was fixed.
I restored the linting rule here.